Save/Preview/BatchEditViewController.swift

```diff
@@ -10,50 +10,58 @@ import UIKit
 import AlignedCollectionViewFlowLayout
 
 class BatchEditViewController: BaseViewController, InfoBoxDelegate {
-
+    
     var assets: [Asset]?
-
+    
     @IBOutlet weak var image1: UIImageView!
     @IBOutlet weak var image2: UIImageView!
     @IBOutlet weak var image3: UIImageView!
-
+    
     @IBOutlet weak var counterLb: UILabel!
     @IBOutlet weak var flagIv: Flag!
-
+    
     @IBOutlet weak var infos: UIView!
     @IBOutlet weak var infosBottom: NSLayoutConstraint?
-
+    private var keyboardConstraint: NSLayoutConstraint?
     private var dh: DarkroomHelper?
-
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-
+        
         let title = MultilineTitle()
         title.title.text = NSLocalizedString("Edit Media Info", comment: "")
         title.subtitle.text = String.localizedStringWithFormat(NSLocalizedString("%u Item(s) Selected", comment: "#bc-ignore!"), assets?.count ?? 0)
         navigationItem.titleView = title
-
+        
         counterLb.text = Formatters.format(assets?.count ?? 0)
         flagIv.isSelected = assets?.reduce(true, { $0 && $1.flagged }) ?? false
-
+        
         setImage(image1, assets?.first)
         setImage(image2, assets?.count ?? 0 > 1 ? assets?[1] : nil)
         setImage(image3, assets?.count ?? 0 > 2 ? assets?[2] : nil)
-
-        // Use the new keyboard layout guide, if available, that's more reliable
-        // than any calculation.
+        
         if #available(iOS 15.0, *) {
+            // Deactivate storyboard constraint if any
             infosBottom?.isActive = false
-            infos.bottomAnchor.constraint(equalTo: infos.keyboardLayoutGuide.topAnchor).isActive = true
+            keyboardConstraint = infos.bottomAnchor.constraint(equalTo: view.keyboardLayoutGuide.topAnchor)
+            
+            
+            keyboardConstraint?.constant = -20
```

Comment:
In this case can we figure out a way to manage this properly as in a generic manner so that this is not happening in the code level at the controller